Skate_Behaviour_IO.m

```diff
@@ -61,10 +69,10 @@
         %time since recording start in ms (integer)
         event_latency = [EEG.event.latency];
         
-        %remove all the extra ones
-        garbage_marker_bolean = strcmp(event_strings,'S  1'); %find strings
-        event_strings(garbage_marker_bolean) = []; %remove ones
-        event_latency(garbage_marker_bolean) = []; %from both
+%         %remove all the extra ones
```

Comment:
doesn't this still need to happen for the first set of subjects?

Reply:
I don't think so, because even though the S1s are extra in those files, the part of this code that matters for this set of subjects is looking for 3, 5, 7 and 9. Even if the 1s are removed as before, that wouldn't interfere with how the means, sd, se... are computed and plotted. 
`        event_markers(strcmp(event_strings,'S  3')) = nStandard;   %standard
        event_markers(strcmp(event_strings,'S  5')) = nTarget;   %target
        event_markers(strcmp(event_strings,'S  7')) = nFalseAlarm;   %false alarm
        event_markers(strcmp(event_strings,'S  9')) = nCorrectResponse;   %correct response`

Reply:
if you don't need the code that remove it, leaving it as commented is confusing, 
this code should be readable by other researchers who look at it, so choose names that are clear and remove any unneeded stuff